src/Tracking/Events/Activated.php

```diff
@@ -0,0 +1,65 @@
+<?php
+declare( strict_types=1 );
+
+namespace Automattic\WooCommerce\GoogleListingsAndAds\Tracking\Events;
+
+/**
+ * This class adds actions to track when the extension is activated.
+ * *
+ *
+ * @package Automattic\WooCommerce\GoogleListingsAndAds\Tracking
+ */
+class Activated extends BaseEvent {
+
+	/**
+	 * The page where activation with a source can occur.
+	 */
+	public const ACTIVATION_PAGE = 'plugin-install.php';
+
+	/**
+	 * The query parameters used to determine activation source details.
+	 */
+	public const SOURCE_PARAMS = [
+		'traffic_source',
```

Comment:
I wonder if it would be more useful to define the commonly-used [UTM parameters](https://en.wikipedia.org/wiki/UTM_parameters#UTM_parameters) here instead?

Reply:
Good call 071cee9af8621bb75551714b6b368ded8f49a74e